.github/workflows/playwright.yml

```diff
@@ -0,0 +1,28 @@
+# disabled because doesn't run on ci yet and don't want to have a failing check
+name: Playwright Tests
+on:
+    push:
+        branches: [main, master]
+    pull_request:
+        branches: [main, master]
+jobs:
+    test:
+        timeout-minutes: 60
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: lts/*
+        # - name: Install dependencies
+        #   run: npm ci
+        # - name: Install Playwright Browsers
+        #   run: npx playwright install --with-deps
+        # - name: Run Playwright tests
+        #   run: npx playwright test
+        # - uses: actions/upload-artifact@v4
+        #   if: always()
```

Comment:
always just returns true? So can remove it?

Reply:
was autogenerated

Reply:
but yeah i guess